### DISCUSSION TITLE
As a user, where should I ask a question about Spack?

### DISCUSSION THREAD

**Discussion by BerengerBerthoul** [spack/spack#25441] [Q&A]
Hi, I have been wondering where users are supposed to ask questions about Spack:
- Here (e.g. Spack github discussions). It would make sense but I see there are not that many questions asked. Plus it is a relatively unknown place of GitHub.
- Issues at https://github.com/spack/spack/issues. I think that this is supposed to be related to real issues (as suggested by the name, even though some projects use it for everything)
- StackOverflow. I think it would be the most natural place to look at. Plus it is well referenced by google. There is a `spack` tag, and there are a few questions (18 at the moment), with some answered by e.g. @tgamblin and @adamjstewart. But the activity is low. As an example I asked two questions with no answer nor comment, seen by less than 20 people.

I guess it would make sense to add a section in the documentation to give an official recommendation on where to ask questions about Spack. The documentation is well thought-out and detailed, but I think we are lacking a central Q&A-style documentation site where the community could interact.

What do you think?

**Comment by adamjstewart** [comment]
There are two other places you're forgetting:

* Slack (probably the best place at the moment)
* Google Group (haven't checked this in several years but I think it's still used)

I check StackOverflow like once per year to see if anyone has asked questions about Spack. I agree that we could really use a central place for Q&A but I'm not sure where. If I was going to suggest a single place, it would be Slack at the moment. But I'm also fine with questions on Issues. I don't visit Discussions very often.
